sealtk/gui/Player.cpp

```diff
@@ -257,13 +261,9 @@ void Player::initializeGL()
 {
   QTE_D();
 
-  connect(this->context(), &QOpenGLContext::aboutToBeDestroyed,
-          [d]() {d->destroyResources();});
```

Comment:
This was in place because the `QDockWidget` destroyed the OpenGL context, and I had to account for that.

Reply:
I would rather get a shared context working.

Reply:
Seems to be working now, though it is not as clean as it could be due to some `QOpenGLTexture` wonkiness.